symbolic-symcache/src/writer.rs

```diff
@@ -416,13 +430,25 @@ where
         Ok(index)
     }
 
-    fn insert_lines(
+    /// Takes an iterator of [`LineInfo`]s and returns a vector containing [`LineRecord`](format::LineRecord)s
+    /// for those lines whose address is between `start_address` and `end_address`.
+    ///
+    /// - If the difference between the addresses of two consecutive
+    /// lines `L1` and `L2` is greater than 255, dummy line records with the same file and line
+    /// information as L1 will be inserted between the two.
```

Comment:
Does this format correctly?  I thought markdown needed these lines indented

Reply:
It is formatted like this, which I think is correct: <img width="967" alt="Bildschirmfoto 2021-02-17 um 12 23 44" src="https://user-images.githubusercontent.com/7622248/108197729-0fea4880-711b-11eb-85f5-c6c61c69e5f5.png">

Reply:
TIL